language/move-lang/src/typing/translate.rs

```diff
@@ -112,6 +112,7 @@ fn check_primitive_script_arg(context: &mut Context, mloc: Loc, idx: usize, ty:
         Type_::bool(loc),
         Type_::address(loc),
         Type_::vector(loc, Type_::u8(loc)),
+        Type_::vector(loc, Type_::vector(loc, Type_::u8(loc))),
```

Comment:
I thought @tnowacki already fixed this. Are you on latest master? Todd?

Reply:
Fixed on master

Reply:
yep, @tnowacki fixed on master. i need to cleanup and rebase this PR 😅